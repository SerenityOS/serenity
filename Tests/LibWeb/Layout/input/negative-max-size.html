<!doctype html><style>
    * {
        font: 16px SerenitySans;
    }
    div {
        border: 1px solid black;
        max-width: -90px;
    }
</style><div>Well, hello friends!</div>