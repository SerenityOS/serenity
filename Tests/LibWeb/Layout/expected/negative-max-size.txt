Viewport <#document> at (0,0) content-size 800x600 children: not-inline
  BlockContainer <html> at (0,0) content-size 800x35.46875 [BFC] children: not-inline
    BlockContainer <body> at (8,8) content-size 784x19.46875 children: not-inline
      BlockContainer <div> at (9,9) content-size 782x17.46875 children: inline
        line 0 width: 147.1875, height: 17.46875, bottom: 17.46875, baseline: 13.53125
          frag 0 from TextNode start: 0, length: 20, rect: [9,9 147.1875x17.46875]
            "Well, hello friends!"
        TextNode <#text>

ViewportPaintable (Viewport<#document>) [0,0 800x600]
  PaintableWithLines (BlockContainer<HTML>) [0,0 800x35.46875]
    PaintableWithLines (BlockContainer<BODY>) [8,8 784x19.46875]
      PaintableWithLines (BlockContainer<DIV>) [8,8 784x19.46875]
        TextPaintable (TextNode<#text>)
